vllm/attention/ops/triton_flash_attention.py

```diff
@@ -507,7 +526,12 @@ def attn_fwd(
     qk_scale = sm_scale * 1.44269504089
     # Q is loaded once at the beginning and shared by all N blocks.
     q = load_fn(Q_block_ptr, True, padded_head, "zero")
-    q = (q * qk_scale).to(Q_block_ptr.type.element_ty)
+    if not USE_FP8:
+        q = (q * qk_scale).to(Q_block_ptr.type.element_ty)
+        acc_scale = 1.0
+    else:
+        qk_scale *= q_scale * k_scale
```

Comment:
Can we compute these products externally? Or do we need to pass both and multiply in every pid?

Reply:
mostly for the sake of clarity; also tried with computing it outside, there wasn't perf difference